docs/decisions/0015-severity-labels.md

```diff
@@ -0,0 +1,93 @@
+# Severity Labels
+
+## Context and Problem Statement
+
+In our current process, we are reviewing our backlog and issues in general during our Backlog Review
+meeting which happened once every 4 weeks in our Release Cycle.
+Thus when a new issue is created, we automatically add the label 'waiting/triage' so we can review
+it during our next Backlog Review. During the Backlog Review meeting, we collectively set a priority
+label (from 0-Highest to 3-Lowest) so we can prioritise our work for the next cycle.
+This is not ideal as an issue can be leftover for weeks before before it's discussed during the
+Backlog Review.
+
+## Solution
+
+To make the review and planning process easier for the Tech Lead, the Engineering Manager and
+Engineers. We would like to introduce a new type of Labels: Severity.
+The goal is to provide a way for the reporter of an Epic/Issue to provide an estimation of the
+impact and help the team to plan an prioritise accordingly.
+
+Distinction between Severity and Priority:
+- The severity field describes the impact of a bug. And the reporter is free to set this up
+- The priority field describes the importance and order in which an issue should be fixed. Priority
+cannot be changed without discussing it with Joao or Vincent.
+
+## Requierements
+
+The severity labels:
+- should be easy to understand and use,
+- be taken into account when prioritising our issues,
+- should be compatible with the Longhorn severity labels,
+- should be adaptable to other Rancher projects,
```

Comment:
Either drop the commas, or ensure that the last item is a period instead of a comma. I'd go with the former.